server/api/usersAtEvents.js

```diff
@@ -0,0 +1,35 @@
+const router = require('express').Router()
+const {User, Event} = require('../db/models')
+const canOnlyBeUsedBy = require('./authMiddleware')
+module.exports = router
+
+router.get(
+  '/:eventId',
+  canOnlyBeUsedBy('admin', 'leader', 'self', 'participant'),
+  async (req, res, next) => {
+    try {
+      const events = await Event.findById(req.params.eventId, {
```

Comment:
We just talked about this.  It works, but feels weird. You should be able to make a more direct query.
```
Event
  .findById(...)
  .then(event=>event.getUsers())
  .then(users=>res.json(users))
  .catch(next)
```